html/elements/select.json

```diff
@@ -58,7 +58,7 @@
         },
         "autocomplete": {
           "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/select#autocomplete",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Attributes/autocomplete",
```

Comment:
dito

Reply:
Fixed in [036c74a](https://github.com/mdn/browser-compat-data/pull/25563/commits/036c74a1d18f9d6d352e80cc72572eb13e43513a).